messaging/MessagingExampleSwift/AppDelegate.swift

```diff
@@ -69,7 +70,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // TODO: Handle data of notification
 
     // Print message ID.
-    print("Message ID: \(userInfo["gcm.message_id"]!)")
+    if userInfo[gcmMessageIDKey] != nil {
```

Comment:
can you use "let" instead of !=nil
if let  gcmMessage = userInfo[gcmMessageIDKey] {
  print gcmMessage